frontend/apps/pwa/src/features/former/validation.ts

```diff
@@ -131,6 +131,24 @@ export const customValidation = {
         message: `Form can have at most ${validationConstants.fields.max} fields`,
       });
     }
+    if (form.formType === FormType.RecipientFormType) {
+      const keyFields = form.fields.filter((field) => {
+        return field.key;
+      });
+      if (keyFields.length !== 1) {
+        errors.push({
+          field: 'fields' as FieldPath<ValidationForm>,
+          message: 'Form needs to have exactly 1 key field',
+        });
+      } else if (keyFields[0].fieldType !== FieldKind.Reference) {
+        console.log('key fields', keyFields);
+
+        errors.push({
+          field: 'fields' as FieldPath<ValidationForm>,
```

Comment:
Not important, but I think `as const` should also work here. It'll turn the type from `string` to `fields`.

Reply:
seems to work, what benefit would the const version have? just shorter? less imports?

Reply:
Yeah, I'm not sure if there's a good reason to use one over the other. I personally use `const`, which as you say uses less imports. But I think both are fine.